demo-assethelper/src/main/java/com/example/MainActivity.java

```diff
@@ -23,7 +23,7 @@ public void onCreate(Bundle savedInstanceState) {
 		ListAdapter adapter = new SimpleCursorAdapter(this, 
 				android.R.layout.simple_list_item_2,
 				employees, 
-				new String[] {Employees.FirstName, Employees.LastName},
+				new String[] {Employees.Firstname, Employees.Lastname},
```

Comment:
It's not camel cases due to the naming inside the sqlite database in the assethelper demo. If you'd name your columns `FIRST_NAME` and `SURNAME` in the table it should still be generated fine.